pootle/apps/pootle_fs/management/commands/init_fs_project.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import logging
+
+from django.core.management import BaseCommand, CommandError
+
+from pootle_project.models import Project
+from pootle_language.models import Language
+
+
+logger = logging.getLogger('pootle.fs')
+
+
+class Command(BaseCommand):
+    help = "Init a new Pootle FS project."
+
+    def add_arguments(self, parser):
+        super(Command, self).add_arguments(parser)
```

Comment:
Not needed according to https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/#django.core.management.BaseCommand.add_arguments